python/chigger/RenderWindow.py

```diff
@@ -262,7 +262,7 @@ def write(self, filename):
         if ext not in writers:
             w = ', '.join(writers.keys())
             msg = "The filename must end with one of the following extensions: {}.".format(w)
-            mooseutils.mooseError(msg)
+            mooseutils.mooseError(msg, dialog=True)
```

Comment:
@aeslaughter I know you like to keep QT out of chigger but hopefully this shouldn't be a problem since we do checks on qt availability before trying to show the dialog. However, the check just tries to import QT which could be successful but if a QApplication doesn't exist yet we still try to show the dialog and will get an error.